internal/cmd/beta/network/create/create.go

```diff
@@ -50,12 +64,20 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				`$ stackit beta network create --name network-1`,
 			),
 			examples.NewExample(
-				`Create an IPv4 network with name "network-1" with DNS name servers and a prefix length`,
-				`$ stackit beta network create --name network-1  --ipv4-dns-name-servers "1.1.1.1,8.8.8.8,9.9.9.9" --ipv4-prefix-length 25`,
+				`Create a routed network with name "network-1"`,
+				`$ stackit beta network create --name network-1 --routed`,
 			),
 			examples.NewExample(
-				`Create an IPv6 network with name "network-1" with DNS name servers and a prefix length`,
-				`$ stackit beta network create --name network-1  --ipv6-dns-name-servers "2001:4860:4860::8888,2001:4860:4860::8844" --ipv6-prefix-length 56`,
+				`Create a network with name "network-1" and no gateway`,
+				`$ stackit beta network create --name network-1 --no-ipv4-gateway`,
+			),
+			examples.NewExample(
+				`Create an IPv4 network with name "network-1" with DNS name servers, a prefix, a gateway and a prefix length`,
+				`$ stackit beta network create --name network-1  --ipv4-dns-name-servers "1.1.1.1,8.8.8.8,9.9.9.9" --ipv4-prefix-length 25 --ipv4-prefix "10.1.2.0/24" --ipv4-gateway "10.1.2.3"`,
+			),
+			examples.NewExample(
+				`Create an IPv6 network with name "network-1" with DNS name servers, a prefix, a gateway and a prefix length`,
+				`$ stackit beta network create --name network-1  --ipv6-dns-name-servers "2001:4860:4860::8888,2001:4860:4860::8844" --ipv6-prefix-length 56 --ipv6-prefix "2001:4860:4860::8888" --ipv6-gateway "2001:4860:4860::8888"`,
 			),
```

Comment:
For some Terraform provider example, the IaaS colleagues said that providing a prefix and a prefix length is redundant, and the prefi length will be ignored if prefix is set. So we should adjust the examples also here

Reply:
updated the examples